src/set_theory/schroeder_bernstein.lean

```diff
@@ -95,8 +100,9 @@ parameters {ι : Type u} {β : ι → Type v}
 @[reducible] private def sets := {s : set (∀ i, β i) |
   ∀ (x ∈ s) (y ∈ s) i, (x : ∀ i, β i) i = y i → x = y}
 
+/-- The cardinals are well-ordered. -/
 theorem min_injective (I : nonempty ι) : ∃ i, nonempty (∀ j, β i ↪ β j) :=
```

Comment:
```suggestion
theorem min_injective (I : nonempty ι) : ∃ i, ∀ j, nonempty (β i ↪ β j) :=
```

this isn't a "suggestion" but more of a question of why we did it the other way round? I think probably whatever the answer is is worth noting; because this isn't the most natural phrasing of it for me

Reply:
maybe as a Π type to make it clear it's not a "true" forall?

Reply:
Eric, you shouldn't get to fond of distinguishing `Π` and `∀` since this distinction is gone in Lean 4.

Reply:
oh, that sounds like for the better! will avoid much beginner confusion... I still wonder why the pi type was done "in this order", though

Reply:
Why not simply `∃ i, ∀ j, β i ↪ β j`, actually?